packages/ui/core-components/src/lib/unsorted/ui/LinkButton.svelte

```diff
@@ -3,11 +3,11 @@
 </script>
 
 <script>
-	export let url;
+	export let link;
```

Comment:
I'm hesitant to make this breaking change

Reply:
I didn't consider that, I was only thinking of normalizing all the href-related props to link similar to our chart drilling feature update.

I will change it back